mesheryctl/cmd/perf.go

```diff
@@ -79,17 +84,26 @@ var perfCmd = &cobra.Command{
 		if len(testURL) > 0 {
 			postData = postData + "\nendpoint_url: " + testURL
 		} else {
-			println("Error: Please enter a TestURL")
+			println("\nError: Please enter a test URL")
 			return
 		}
 
+		// Methord to check if the entered Test URL is valid or not
+		var validURL bool = govalidator.IsURL(testURL)
+
+		if (validURL == false) {
+			println("\nError: Please enter a valid test URL")
```

Comment:
This validated doesn’t happen to give specific reasons as to why a given url isn’t valid, does it?  Something specific that we could pass along to the user?

Reply:
> This validated doesn’t happen to give specific reasons as to why a given url isn’t valid, does it? Something specific that we could pass along to the user?

It is a regex check to see if the URL is valid or not. This will prevent the user from entering random strings while performing checks. Like if I enter `ABC` in URL argument , it is accepted as a valid URL.